remotion/compositions/showcases/devfestNantes/Details.tsx

```diff
@@ -0,0 +1,58 @@
+import {loadFont} from '@remotion/google-fonts/LeagueSpartan';
+import {interpolate, spring, useCurrentFrame, useVideoConfig} from 'remotion';
+
+import {TalkDetails} from '../../../design/molecules/TalkDetails';
+
+const {fontFamily} = loadFont();
+
+export const Details: React.FC<{
+	date: string;
+	time: string;
+	location?: string;
+}> = ({date, time, location}) => {
+	const frame = useCurrentFrame();
+	const {fps} = useVideoConfig();
+
+	const drop = spring({
+		frame: frame - 40,
+		from: -40,
+		to: 20,
+		fps,
+		durationInFrames: 40,
+	});
+	const opacity = spring({
+		frame: frame - 40,
+		from: 0,
+		to: 1,
+		fps,
+		durationInFrames: 40,
+	});
+	const textUnblur = interpolate(frame - 40, [0, 18], [5, 0], {
```

Comment:
```suggestion
	const textUnblur = interpolate(frame, [0, 18], [5, 0], {
```

We could put 70frames in sequence definition ;)